src/test/java/teammates/test/cases/action/AdminActivityLogPageActionTest.java

```diff
@@ -379,7 +379,7 @@ public void removeAndRestoreManyLogs() {
                 today.getTime(), LOG_MESSAGE_INTERVAL_MANY_LOGS);
     }
 
-    @Test(groups = "manyLogs", dependsOnGroups = "typicalLogs")
+    @Test(groups = "manyLogs", priority = 2)
```

Comment:
I would expect developers to know the `priority` in TestNG and thus they will quickly know that this test will fun after test without priority (default highest priority = 0).

In fact, `priority` is widely used for test cases in Teammates (action test, driver test, etc.).